.gitignore

```diff
@@ -0,0 +1 @@
+.idea
```

Comment:
what is this?

Reply:
This is something golang creates, should be ignored

Reply:
it's unclear to me why golang would put files in this repo. Can you elaborate?

Reply:
Sorry, goland. But as it is popular one with vs it will be nice to ignore it

Reply:
Please use your own global gitignore settings to exclude files. Not everyone uses vs or pycharm.